_pages/Long-term-support/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+permalink: /Long-term-support/
```

Comment:
The other pages are in small letters, no capital. For consistency, I suggest using long-term-support instead of Long-term-support.

Reply:
modified to lower case